crates/network-scanner-net/Cargo.toml

```diff
@@ -6,9 +6,23 @@ edition = "2021"
 publish = false
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
 anyhow = "1.0.79"
+crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
+parking_lot = "0.12.1"
+polling = "3.3.2"
 socket2 = { version = "0.5.5", features = ["all"] }
-tokio = { version = "1.35.1", features = ["io-util","rt","sync"] }
+thiserror = "1.0.56"
 tracing = "0.1.40"
+
+[dev-dependencies]
+tracing-subscriber = "0.3.18"
+tokio = { version = "1.35.1", features = [
+    "rt",
+    "sync",
+    "macros",
+    "rt-multi-thread",
+    "time",
+    "net"
+] }
+trust-dns-resolver = "0.23.2"
```

Comment:
issue: `trust-dns` project has been rebranded to "Hickory DNS" and has been moved to the https://github.com/hickory-dns/hickory-dns organization and repo

question: Do we really need to bring in this crate just for the example? Can’t we just use the built-in [`ToSocketAddrs` trait](https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html)? There is also [an async version provided by Tokio](https://docs.rs/tokio/latest/tokio/net/trait.ToSocketAddrs.html).

suggestion: Remove the dependency if it’s not absolutely necessary.

Reply:
you are right, updated